lib/workers/dependency.rb

```diff
@@ -47,6 +43,30 @@ def get_results(dependencies)
     end
   end
 
+  def generate_stats(locations)
+    results = {}
+    status_types.each do |type|
+      num = stats(type, locations)
+      results[type] = num
+      results[:outdated_total] = results[:outdated_total].to_i + num unless [:ok, :unknown].include?(type)
```

Comment:
This means `:unknown`s are not being treated as outdated. Does that make sense?

Reply:
That part makes sense, because conceptually it is unknown whether the dependency is outdated or not! Practically though, we could introduce a flag to generate a distinct badge for when a project contains unknown dependencies.